code/style.css

```diff
@@ -117,24 +158,90 @@ main {
 .input-wrapper form {
   width: 100%;
   display: flex;
+  flex-direction: column;
 }
 
 button {
-  background-color: #0026ff;
+  background-color:rgb(226, 194, 185);
   color: white;
   border: none;
   border-radius: 4px;
   padding: 16px 40px;
   margin-right: 4px;
   font-size: 16px;
   line-height: 26px;
-  font-family: 'Montserrat';
+  font-family: "Montserrat";
   font-weight: 500;
   cursor: pointer;
   transition: all 0.3s ease;
+  border-radius: 10px;
+  border:1px solid #fff;
 }
 
 button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
+  transform: scale(1.1,1.1);
+
 }
+#smallBtn, #mediumBtn,#largeBtn {
```

Comment:
generally its consider that class name should be used for styling and Id should be used in JS. so maybe consider it in your future projects.